src/components/Calendar.jsx

```diff
@@ -33,28 +41,18 @@ function Calendar() {
   const [clickedEvent, setClickedEvent] = useState([]);
   const [showEventDetails, setShowEventDetails] = useState(false);
   const [eventDetails, setEventDetails] = useState(false);
-  const [aspectRatio, setAspectRatio] = useState(
-    window.outerWidth > window.innerHeight
-      ? 1.35
-      : window.innerWidth / window.innerHeight
-  );
-  const [initialView, setInitialView] = useState(
-    window.outerWidth > 600 ? 'dayGridMonth' : 'timeGridWeek'
-  );
+  const [aspectRatio, setAspectRatio] = useState(getAspectRatio());
+  const [initialView, setInitialView] = useState(getInitialView());
 
   useEscKey(() => setShowEventDetails(false));
 
   const [popupPosition, setPopupPosition] = useState({});
 
   const windowResize = () => {
-    setAspectRatio(
-      window.outerWidth > window.innerHeight
-        ? 1.35
-        : window.innerWidth / window.innerHeight
-    );
-    setInitialView(window.outerWidth > 600 ? 'dayGridMonth' : 'timeGridWeek');
+    setAspectRatio(getAspectRatio());
+    setInitialView(isMinWidth() ? 'dayGridMonth' : 'timeGridWeek');
     setShowEventDetails(false);
-    window.outerWidth < 600 && setPopupPosition({ left: 0, top: 0 });
+    !isMinWidth() && setPopupPosition({ left: 0, top: 0 });
```

Comment:
Since the build creates a server side rendering of the site, `window` is not available and fails the build.